array-manipulation-3/src/unique.js

```diff
@@ -0,0 +1,25 @@
+/* exported unique */
+
+/*
+- check if input array is empty -> if yes return empty array []
+- create new empty array uniqueArray
+- push first item of the input array into uniqueArray because it's always unique
+- use for loop to iterate through each item of the array starting at index 1
+- check if array[i] is included in the uniqueArray
+- if not, push item onto uniqueArray
+- return uniqueArray
+*/
+
+function unique(array) {
+  if (array.length === 0) {
+    return [];
+  }
+  var uniqueArray = [];
+  uniqueArray.push(array[0]);
+  for (var i = 1; i < array.length; i++) {
+    if (!Array.prototype.includes.call(uniqueArray, array[i])) {
```

Comment:
Again, no need for the `call` method here. I don't know where you picked this up, but it is _very rarely_ necessary